### PR TITLE
Filepicker specs

### PR DESCRIPTION
**Pull request by giladgrush**

### CODE REVIEW

File: docs/filepicker/filepicker.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+| Name | Type | Default | Required | Description |
+| -- | -- | -- | -- | -- |
+| id | string | null | no | Unique identifier |
+| name | string | null | no | Name of the value of the content (passed to hidden input element) |
+| className | string | null | no | Space separated list of the classes of the element |
+| style | Object | null | no | CSS styling declarations to be applied to the element |
+| children | React.ReactNode | null | no | The elements to render in the tooltip |
+| tabIndex | number | 0 | no | Indicates if element participates in keyboard navigation |
+| accept | string \| array | null | no | Specifies the type of file/s the user is allowed to pick. Extensions and mimetypes are supported ('image/*', ['image/\*', '.pdf', 'video/mp4') |
+| files | HTMLInputElement.files | null | no | An object that represents the files property of the native HTML \<input> element |
```
Comment (tobich) [inline docs/filepicker/filepicker.md]: Maybe this should be `value` and onFileSelected should become `onChange`? After all, this is a form input component.

File: docs/filepicker/filepicker.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+| onFileSelected | Function(file) | NOOP | no | Called whenever user selects a file. The callback has a file parameter object that contains the filename, mimetype, size in bytes, and source. |
+| maxSize | number | null | no | Maximum size of file to upload specified in bytes |
+| minSize | number | null | no | Minimum size of file to upload specified in bytes |
+| maxFiles | number | 1 | no | Number of files which can be selected in the component |
```
Comment (tobich) [inline docs/filepicker/filepicker.md]: Missing `enctype`. Its default should probably be `"multipart/form-data"`

Comment (tobich) [inline docs/filepicker/filepicker.md]: I suggest replacing `maxFiles` with `multiple` (like the HTML version). `maxSize`, `minSize` and `maxFiles` can be handled by custom validation of the `files` (or `value`) property outside of the component (controlled props).

File: docs/filepicker/filepicker.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+### Touch
+
+Not relevant
+
+## Examples
+
+* Create a demo that allows the user to pick multiple files of type image. It should show the files in a list and the user should be able to remove the files. See image below:
+
+    ![Image of filepicker demo](./assets/demofilepicker.png)
```
Comment (tobich) [inline docs/filepicker/filepicker.md]: Shouldn't this be the whole component? The current spec is just a thin veneer over the native `<input type="file"/>`. We can supply the list of files with `itemRenderer` and the "delete" button. (The native input displays filename if single file selected, and the "x files selected" when multiple are.)

File: docs/filepicker/filepicker.md (hunk 2 above)
Comment (giladgrush) [reply docs/filepicker/filepicker.md]: No need

Comment (giladgrush) [reply docs/filepicker/filepicker.md]: Sounds good!

File: docs/filepicker/filepicker.md (hunk 3 above)
Comment (giladgrush) [reply docs/filepicker/filepicker.md]: Talked about this, keep it as it currently is

File: docs/filepicker/filepicker.md (hunk 1 above)
Comment (giladgrush) [reply docs/filepicker/filepicker.md]: Let's discuss, seems that a change may be required in the automatic *controlled* mechanism

**Pull request merged**